app/views/rich_text_reactions/_emoji_selector.html.erb

```diff
@@ -0,0 +1,32 @@
+<%#
+Locals required:
+- rich_text_id: Id of the ActionText::RichText that the new RichTextReaction belongs to
+
+Helpers used:
+- app/helpers/emojis_helper
+%>
+
+<div class="relative">
+  <details class="emoji-selection inline-block align-middle">
+    <summary class="list-marker-none cursor-pointer">
+      <%= render inline: Rails.root.join('app/assets/images/smiley_face.svg').read %>
```

Comment:
An `image_tag` tag could be used here instead

Reply:
Also, is there a reason why you went with an image? The smiley face emoji could also be displayed as text